cleaners/opera.xml

```diff
@@ -29,13 +37,20 @@ Notes about Opera
   <running type="exe" os="linux">opera</running>
   <running type="exe" os="windows">opera.exe</running>
   <var name="base">
-    <value>%UserProfile%\AppData\Roaming\Opera Software\Opera Stable</value>
-    <value>$XDG_CONFIG_HOME/opera</value>
+    <!-- Lets make it "universal"... Was:
+    <value os="windows">%AppData%\Opera Software\Opera Stable</value>
+    Is now: -->
+    <value os="windows">%AppData%\Opera Software\Opera*</value>
```

Comment:
I tested today on Windows 10 with Opera beta (Opera version 68). It made a directory `Opera Software\Opera Next`.

When the username was ASCII, this worked. When the username had Unicode, this failed unless I edited the cleaner to replace the glob with the exact name.